test_run.py

```diff
@@ -31,6 +34,19 @@ def retrieve_test_list(
 
 
 if __name__ == '__main__':
+    help_text = dedent(
+        """
+        The tests "test_benchmark.py" and "test_commit_format.py" work properly
```

Comment:
This is starting too abruptly I think. I would begin with that the script does, i.e. lets you run all the tests locally in one shot or something like that.